Gems/EMotionFX/Code/EMotionFX/Exporters/ExporterLib/Exporter/SkeletalMotionExport.cpp

```diff
@@ -56,12 +56,56 @@ namespace ExporterLib
         }
     }
 
+    void SaveRootMotionExtractionData(MCore::Stream* file, EMotionFX::Motion* motion, MCore::Endian::EEndianType targetEndianType)
+    {
+        if (!motion->GetRootMotionExtractionData())
+        {
+            return;
+        }
+
+        AZ::SerializeContext* serializeContext = nullptr;
+        AZ::ComponentApplicationBus::BroadcastResult(serializeContext, &AZ::ComponentApplicationBus::Events::GetSerializeContext);
+        if (!serializeContext)
+        {
+            AZ_Error("EMotionFX", false, "Can't get serialize context from component application.");
```

Comment:
nothing action items to add here right? If the serialize context doesn't exist, there's something terrible happened

Reply:
Yea.. Not sure if those check are needed honestly, maybe in some automation test that could happen.